xwiki-platform-core/xwiki-platform-attachment/xwiki-platform-attachment-ui/src/main/resources/XWiki/AttachmentSelector.xml

```diff
@@ -529,17 +529,17 @@ $xwiki.jsx.use($attachmentPickerDocName)
   }
 
   /** Handles the gallery buttons directly in the current page without reloading the window. */
-  XWiki.AttachmentPicker = Class.create({
+  class AttachmentPicker {
```

Comment:
I'm worried that some extensions might use the ``XWiki.AttachmentPicker``. It's good that you defined it as a standard class, but I'd still expose it on ``XWiki`` for backwards compatibility. Or at least provide a way for old code to migrate to the new class.